src/core/bittorrent/torrenthandle.h

```diff
@@ -178,9 +178,46 @@ namespace BitTorrent
         qlonglong pieceLength() const;
         qlonglong wastedSize() const;
         QString currentTracker() const;
-        QString actualSavePath() const;
-        QString savePath() const;
-        QString rootPath() const;
+
+        // 1. savePath() - the path where all the files and subfolders of torrent are stored (as always).
+        // 2. rootPath() - absolute path of torrent file tree (save path + first item from 1st torrent file path).
+        // 3. contentPath() - absolute path of torrent content (root path for multifile torrents, absolute file path for singlefile torrents).
+        //
+        // These methods have 'actual' parameter (default to false) which allow to get current or target path variant.
```

Comment:
If I read correctly:

These methods have 'actual' parameter: when false, returns the path where files are stored (current path); when true, returns the path where files should be stored (target path).